extras/nightly-container/provision.yml

```diff
@@ -111,3 +113,35 @@
       args:
         warn: no
 
+    - name: Configure gluster-exporter to use glusterd2
+      replace:
+        path: /etc/gluster-exporter/gluster-exporter.toml
+        regexp: '^gluster-mgmt\s*=.*'
+        replace: 'gluster-mgmt = "glusterd2"'
+
+    - name: Configure gluster-exporter to use glusterd2 workdir
+      replace:
+        path: /etc/gluster-exporter/gluster-exporter.toml
+        regexp: '^glusterd-dir\s*=.*'
+        replace: 'glusterd-dir = "/var/lib/glusterd2"'
+
+   - name: Create override directory for gluster-exporter.service
+      file:
+        path: /etc/systemd/system/gluster-exporter.service.d
+        state: directory
+
+    # This makes systemd pass environment variables set for
+    # glusterd2/gluster-exporter by kubernetes
+    - name: Enable PassEnvironment override for gluster-exporter.service
+      ini_file:
+        path: /etc/systemd/system/gluster-exporter.service.d/override.conf
+        section: Service
+        option: PassEnvironment
+        # PassEnvironment requires that each variable be mentioned individually
+        # More variables will need to be added here as required
+        value: GD2_ENDPOINTS
+
+    - name: Enable gluster-exporter.service
+      command: systemctl enable gluster-exporter.service
```

Comment:
You should be using the service module instead of the command module. Any reason not to?

Reply:
referred other instance in the same file(Line 109)